src/core/memory.cc

```diff
@@ -2271,7 +2271,7 @@ void configure(std::vector<resource::memory> m, bool mbind, std::optional<std::s
 }
 
 statistics stats() {
-    return statistics{0, 0, 0, 1 << 30, 1 << 30, 0, 0, 0, 0, 0};
+    return statistics{0, 0, 0, 1 << 30, 1 << 30, 0, 0, 0, 0, 0, 0};
 }
```

Comment:
this change should be moved to patch "Introduce a failed allocations stat"

Reply:
@raphaelsc - good catch: done in force 7c354ae.